templates/invoice/invoice.tex.ep

```diff
@@ -36,7 +37,7 @@ patrons@omniosce.org
 \end{minipage}
 
 \vspace{4ex}
-<%== localtime->strftime('Olten, %e %B %Y') %>
+<%== Time::Piece->new($date)->strftime('Olten, %e %B %Y') %>
```

Comment:
if you use Time::Piece I would also `use Time::Piece`

Reply:
there is a `use Time::Piece` in line # 1